build/configs/imxrt1050-evk/scripts/memory.ld

```diff
@@ -95,15 +95,15 @@ MEMORY
 
   kdtcm   (rwx) : ORIGIN = 0x20000000, LENGTH = 64K
   udtcm   (rwx) : ORIGIN = 0x20010000, LENGTH = 16K
-  kocram (rwx) : ORIGIN = 0x20200000, LENGTH = 64K
+  kocram (rwx) : ORIGIN = 0x20200000, LENGTH = 256K
   uocram (rwx) : ORIGIN = 0x20240000, LENGTH = 64K
 }
 
 __kflash_segment_start__  = ORIGIN(kflash);
 __kflash_segment_size__   = LENGTH(kflash);
 __uflash_segment_start__  = ORIGIN(uflash);
 __uflash_segment_size__   = LENGTH(uflash);
-__ksram_segment_start__   = ORIGIN(kdtcm);
-__ksram_segment_size__    = LENGTH(kdtcm);
-__usram_segment_start__   = ORIGIN(udtcm);
-__usram_segment_size__    = LENGTH(udtcm);
+__ksram_segment_start__   = ORIGIN(kocram);
```

Comment:
if kdtcm is configured, do we need separate ld script?

Reply:
Yes. Either we need a separate ld script. Or.. we can change the values here. 
However, it is likely that we will move to SD ram.. and we might not use the dtcm or ocram. 
For now, i think we can keep these changes.